docs/standard/microservices-architecture/container-docker-introduction/docker-defined.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: What is Docker? | Microsoft Docs 
+title: What is Docker?
```

Comment:
btw --- That description metadata ... Does it conform to guidance? I thought they wanted one (or two) just plain action-style sentences 115-145 chars (where possible). (??)